### PR TITLE
[`InstructBlip`] Fix instructblip slow test

### PR DESCRIPTION
**Pull request by younesbelkada**
# What does this PR do?

Fixes the current failing daily CI test: https://github.com/huggingface/transformers/actions/runs/5675853423/job/15381807774 let's make the daily CI happy!

Ran the test on the latest docker image and the test now pass with these values.

cc @sgugger @ydshieh

### CODE REVIEW

**Review by sgugger — APPROVED** [review]
Thanks for the fix!

File: tests/models/instructblip/test_modeling_instructblip.py [hunk 1]
```diff
@@ -537,9 +537,10 @@ def test_inference_vicuna_7b(self):
             logits = model(**inputs).logits
 
         expected_slice = torch.tensor(
-            [[-3.5410, -12.2812, 8.2812], [-5.2500, -12.0938, 7.8398], [-4.1523, -13.8281, 9.0000]],
+            [[-3.5020, -12.3281, 8.4453], [-5.1406, -11.9609, 7.8711], [-4.0430, -13.4375, 9.1172]],
             device=torch_device,
         )
+        print(logits[0, :3, :3].float())
```
Comment (ydshieh) [inline tests/models/instructblip/test_modeling_instructblip.py:543]: not `print` here 😅

Comment (younesbelkada) [inline tests/models/instructblip/test_modeling_instructblip.py:543]:
```suggestion
```

**Review by ydshieh — APPROVED** [review]
Thanks for the fix. Before merge, please remove `print` statement 🙏

**Pull request merged**